tests/doc_task_test.py

```diff
@@ -461,3 +462,25 @@ class MockDbtSugarConfig:
     )
     doc_task.document_columns(doc_task.dbt_definitions)
     doc_task.dbt_definitions = {"columnA": "newDescriptionA", "columnB": "descriptionB"}
+
+
+@pytest.mark.parametrize(
+    "model_name, path_model, is_schema",
+    [
+        pytest.param(
+            "my_first_dbt_model",
+            Path("tests/test_dbt_project/dbt_sugar_test/models/example/schema.yml"),
+            False,
+        ),
+        pytest.param(
```

Comment:
so now you're using this wrapper you can pass an `id=` variable that describes a bit the scenario